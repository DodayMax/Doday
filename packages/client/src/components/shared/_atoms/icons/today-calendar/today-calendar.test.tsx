import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('TodayCalendar', () => {
  it('renders correctly without props', () => {
    const todayCalendar = shallow(<Icons.TodayCalendar />);

    expect(todayCalendar).toMatchSnapshot();
  });
});
