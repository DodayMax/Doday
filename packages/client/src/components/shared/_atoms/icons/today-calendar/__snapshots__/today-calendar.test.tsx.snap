// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodayCalendar renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <defs>
          <path
            d="M20 8V5h-2v1h-2V5H8v1H6V5H4v3h16zm0 2H4v10h16V10zm-2-7h2a2 2 0 0 1 2 2v15a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h2V2h2v1h8V2h2v1zm-7 12.586l4.293-4.293 1.414 1.414L11 18.414l-3.707-3.707 1.414-1.414L11 15.586z"
            id="calendar-1"
          />
        </defs>,
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <mask
            fill="#fff"
            id="calendar-2"
          >
            <use
              xlinkHref="#calendar-1"
            />
          </mask>
          <g
            fill="black"
            mask="url(#calendar-2)"
          >
            <path
              d="M0 0h24v24H0z"
            />
          </g>
        </g>,
      ],
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
      "xmlnsXlink": "http://www.w3.org/1999/xlink",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <path
            d="M20 8V5h-2v1h-2V5H8v1H6V5H4v3h16zm0 2H4v10h16V10zm-2-7h2a2 2 0 0 1 2 2v15a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h2V2h2v1h8V2h2v1zm-7 12.586l4.293-4.293 1.414 1.414L11 18.414l-3.707-3.707 1.414-1.414L11 15.586z"
            id="calendar-1"
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M20 8V5h-2v1h-2V5H8v1H6V5H4v3h16zm0 2H4v10h16V10zm-2-7h2a2 2 0 0 1 2 2v15a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h2V2h2v1h8V2h2v1zm-7 12.586l4.293-4.293 1.414 1.414L11 18.414l-3.707-3.707 1.414-1.414L11 15.586z",
            "id": "calendar-1",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        "type": "defs",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />,
            <mask
              fill="#fff"
              id="calendar-2"
            >
              <use
                xlinkHref="#calendar-1"
              />
            </mask>,
            <g
              fill="black"
              mask="url(#calendar-2)"
            >
              <path
                d="M0 0h24v24H0z"
              />
            </g>,
          ],
          "fill": "none",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M0 0h24v24H0z",
              "fill": "none",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <use
                xlinkHref="#calendar-1"
              />,
              "fill": "#fff",
              "id": "calendar-2",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "xlinkHref": "#calendar-1",
              },
              "ref": null,
              "rendered": null,
              "type": "use",
            },
            "type": "mask",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <path
                d="M0 0h24v24H0z"
              />,
              "fill": "black",
              "mask": "url(#calendar-2)",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M0 0h24v24H0z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            "type": "g",
          },
        ],
        "type": "g",
      },
    ],
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <defs>
            <path
              d="M20 8V5h-2v1h-2V5H8v1H6V5H4v3h16zm0 2H4v10h16V10zm-2-7h2a2 2 0 0 1 2 2v15a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h2V2h2v1h8V2h2v1zm-7 12.586l4.293-4.293 1.414 1.414L11 18.414l-3.707-3.707 1.414-1.414L11 15.586z"
              id="calendar-1"
            />
          </defs>,
          <g
            fill="none"
            fillRule="evenodd"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <mask
              fill="#fff"
              id="calendar-2"
            >
              <use
                xlinkHref="#calendar-1"
              />
            </mask>
            <g
              fill="black"
              mask="url(#calendar-2)"
            >
              <path
                d="M0 0h24v24H0z"
              />
            </g>
          </g>,
        ],
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
        "xmlnsXlink": "http://www.w3.org/1999/xlink",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <path
              d="M20 8V5h-2v1h-2V5H8v1H6V5H4v3h16zm0 2H4v10h16V10zm-2-7h2a2 2 0 0 1 2 2v15a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h2V2h2v1h8V2h2v1zm-7 12.586l4.293-4.293 1.414 1.414L11 18.414l-3.707-3.707 1.414-1.414L11 15.586z"
              id="calendar-1"
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M20 8V5h-2v1h-2V5H8v1H6V5H4v3h16zm0 2H4v10h16V10zm-2-7h2a2 2 0 0 1 2 2v15a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h2V2h2v1h8V2h2v1zm-7 12.586l4.293-4.293 1.414 1.414L11 18.414l-3.707-3.707 1.414-1.414L11 15.586z",
              "id": "calendar-1",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          "type": "defs",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />,
              <mask
                fill="#fff"
                id="calendar-2"
              >
                <use
                  xlinkHref="#calendar-1"
                />
              </mask>,
              <g
                fill="black"
                mask="url(#calendar-2)"
              >
                <path
                  d="M0 0h24v24H0z"
                />
              </g>,
            ],
            "fill": "none",
            "fillRule": "evenodd",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M0 0h24v24H0z",
                "fill": "none",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <use
                  xlinkHref="#calendar-1"
                />,
                "fill": "#fff",
                "id": "calendar-2",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "xlinkHref": "#calendar-1",
                },
                "ref": null,
                "rendered": null,
                "type": "use",
              },
              "type": "mask",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <path
                  d="M0 0h24v24H0z"
                />,
                "fill": "black",
                "mask": "url(#calendar-2)",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "d": "M0 0h24v24H0z",
                },
                "ref": null,
                "rendered": null,
                "type": "path",
              },
              "type": "g",
            },
          ],
          "type": "g",
        },
      ],
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
